package exception;

public class TryWithMultipleCatchExample {

	public static void main(String[] args) {
		
		try
		{
			// Here i am writing some logic
		}
		catch (ArithmeticException e) {
			
		}
	}
}
